src/Languages/Editor/Impl/ContainedLanguage/ContainedLanguageHost.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information
+
+using Microsoft.Languages.Editor.Composition;
+using Microsoft.Languages.Editor.Services;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.Languages.Editor.ContainedLanguage {
+    /// <summary>
+    /// Helper class that simplifies location of contained language host
+    /// for contained (secondary) languages. 
+    /// </summary>
+    public static class ContainedLanguageHost {
```

Comment:
So it is kind of helper for `IContainedLanguageHostProvider`?

---
In reply to: [64441532](https://github.com/Microsoft/RTVS/pull/1812#discussion_r64441532) [](ancestors = 64441532)